lib/target.js

```diff
@@ -521,6 +521,7 @@ class Target {
                 })
               );
             } else {
+              console.info(`Identity is not available: "${opts.identity_display}"`);
```

Comment:
Having had this while debugging would have saved me a ton of time, so I'll leave it in :)